ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource

```diff
@@ -97,10 +97,10 @@ Create Pipeline Run    # robocop: disable
 Click Action From Actions Menu
     [Documentation]    Click an action from Actions menu (3-dots menu on the right)
     [Arguments]    ${pipeline_name}    ${action}
-    Wait Until Page Contains Element    xpath://table//tr[td[a[text()='${pipeline_name}']]]//td[contains(@class, 'pf-c-table__action')]/div    # robocop: disable
-    Click Element    xpath://table//tr[td[a[text()='${pipeline_name}']]]//td[contains(@class, 'pf-c-table__action')]/div    # robocop: disable
-    Wait Until Page Contains Element    //table//tr[td[a[text()='${pipeline_name}']]]//td[contains(@class, 'pf-c-table__action')]/div/ul/li/button[contains(text(), '${action}')]    # robocop: disable
-    Click Element    //table//tr[td[a[text()='${pipeline_name}']]]//td[contains(@class, 'pf-c-table__action')]/div/ul/li/button[contains(text(), '${action}')]    # robocop: disable
+    Wait Until Page Contains Element    xpath://table//tr[td[*[a[text()='${pipeline_name}']]]]//td[contains(@class, 'pf-c-table__action')]/div    # robocop: disable
```

Comment:
Instead of adding `# robocop: disable` on each line, could you add `# robocop: disable=line-too-long` on top of the keyword title, like in [this example](https://github.com/red-hat-data-services/ods-ci/blob/a13a932465fc480cbd6f621943eea5ba9b0e7855/ods_ci/tests/Resources/Page/Operators/OpenShiftPipelines.resource#L18)

Reply:
Done